src/getPdfMetadata.ts

```diff
@@ -1,28 +1,31 @@
-import PdfMetadata from "./pdfMetadata";
+import { PdfLib, PdfMetadata } from "./types";
 import { capitalizeFirstLetter } from "./helperFunctions";
-const pdfLib = require('pdf-to-text');
+//@ts-expect-error no type declarations for this module
+import pdfLib from "pdf-to-text";
 
 // Placeholder until there's a front-end where this param can be input
-const pdfFilePath = "../dhl-handbuch-funktion-retoure-v7-122019.pdf"
+const pdfFilePath = "../dhl-handbuch-funktion-retoure-v7-122019.pdf";
 
 // Bad file error test
 // const pdfFilePath = "2f";
 
 // Getting the .pdf metadata out of the callback function
-const getPdfMetadata = async(pdfLib: any, pdfFilePath: string): Promise<PdfMetadata> => {
-  return new Promise((resolve, reject) => {
-    const metadata:PdfMetadata = pdfLib.info(pdfFilePath, (err: string, results: PdfMetadata) => {
-      if (err) {
-        console.log(capitalizeFirstLetter(err))
-        reject(capitalizeFirstLetter(err))
-      }
-      resolve(results)
-    });
-    return metadata;
-  });
-}
+const getPdfMetadata = async (
+   pdfLib: PdfLib,
```

Comment:
Awesome work using injection here. Let's be a bit more specific with the type though. `any` is a bit to loose